packages/frontend/app/services/channels/-utils/vote-sorter.ts

```diff
@@ -0,0 +1,19 @@
+import VoteChain from 'emberclear/models/vote-chain';
+import Identity from 'emberclear/models/identity';
+
+export function generateSortedVote(vote: VoteChain): Uint8Array {
+  let toReturn = [
+    toSortedPublicKeys(vote.remaining),
+    toSortedPublicKeys(vote.yes),
+    toSortedPublicKeys(vote.no),
+    vote.target.publicKey,
+    vote.action,
+    vote.key.publicSigningKey,
+    vote.previousVoteChain?.signature,
+  ];
+  return new TextEncoder().encode(JSON.stringify(toReturn));
```

Comment:
You may be interested in this file: https://github.com/NullVoxPopuli/emberclear/blob/master/packages/frontend/app/utils/string-encoding.ts#L19

Reply:
You may be interested in this file: https://github.com/NullVoxPopuli/emberclear/blob/master/packages/frontend/app/utils/string-encoding.ts#L19

Reply:
The nacl hasher and signing unfortunately needs to take in Uint8Arrays, so I think we will need to encode them into that regardless.

Reply:
We could extract the object->Uint8Array into that other file though, so it could be universal across the board.